concepts/booleans/about.md

```diff
@@ -1 +1,75 @@
 # The Boolean Type
+
+In Cairo, the Boolean type is essential for logical operations and conditional statements. Booleans in Cairo are represented as one `felt252` in size, ensuring efficient use of space while maintaining the simplicity of true/false logic.
+
+The Boolean type in Cairo is specified using the keyword `bool`. This enables the declaration and manipulation of Boolean variables within the language. For instance, consider the following example:
+
+```rust
+fn main() {
+    let t = true;
+
+    let f: bool = false; // with explicit type annotation
+}
+```
+
+In this example, two Boolean variables are declared. The variable $t$ is assigned the value true implicitly, while the variable $f$ is explicitly annotated with the bool type and assigned the value false.
+
+One important aspect to note is that when declaring a bool variable in Cairo, it is mandatory to use either the true or false literals as the value. This means that integer literals, such as $0$ or $1$, cannot be used as substitutes for false or true. The strict enforcement of this rule ensures type safety and prevents potential logical errors in smart contract development.
+
+
+The following code snippet demonstrates how to use Boolean types in Cairo. It shows the declaration of Boolean variables, assignment of Boolean expressions, and usage of assertions to verify the correctness of the Boolean logic.
+
+```rust
```

Comment:
This is a useful code example, and it can be expanded in the following way:
- before the example, add a section that lists boolean operators `||`, `&&` and `!` and provides a brief description for each
- include these operators in the example below
- the subtitles and line-by-line explanations after the example can be removed, as the example is clear enough; alternatively, you can add short descriptions in inline comments